driver/bpf/fillers.h

```diff
@@ -1066,16 +1107,17 @@ FILLER(sys_getrlimit_setrlrimit_x, true)
 		max = -1;
 	}
 
-	/*
-	 * cur
-	 */
+	/* Parameter 2: cur (type: PT_ERRNO) */
 	res = bpf_push_s64_to_ring(data, cur);
 	CHECK_RES(res);
 
-	/*
-	 * max
-	 */
-	return bpf_push_s64_to_ring(data, max);
+	/* Parameter 3: max (type: PT_ERRNO) */
+	res = bpf_push_s64_to_ring(data, max);
+	CHECK_RES(res);
+	
+	/* Parameter 4: resource (type: PT_ERRNO) */
+	unsigned long resource = bpf_syscall_get_argument(data, 0);
```

Comment:
```suggestion
	uint32_t resource = bpf_syscall_get_argument(data, 0);
```

Reply:
Same in all other drivers

Reply:
Have a doubt:  https://github.com/falcosecurity/libs/pull/1348#discussion_r1368271615